app/src/main/java/mozilla/lockbox/view/ItemDetailFragment.kt

```diff
@@ -32,9 +35,13 @@ class ItemDetailFragment : BackableFragment(), ItemDetailView {
         val view = inflater.inflate(R.layout.fragment_item_detail, container, false)
         setupBackable(view)
 
+        addressLayout = view!!.inputHostname
         return view
     }
 
+    override val addressLayoutClicks: Observable<Unit>
+        get() = addressLayout!!.clicks()
```

Comment:
Use 

```kt
get() = view!!.inputHostname.clicks()
```

Also: I don't understand what `addressLayoutClicks` is referring to. Suggest changing to `hostnameClicks`.

Reply:
fixed